cmd/kube-apiserver/app/server.go

```diff
@@ -207,6 +208,7 @@ func (s *APIServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.EtcdPathPrefix, "etcd-prefix", s.EtcdPathPrefix, "The prefix for all resource paths in etcd.")
 	fs.StringSliceVar(&s.CorsAllowedOriginList, "cors-allowed-origins", s.CorsAllowedOriginList, "List of allowed origins for CORS, comma separated.  An allowed origin can be a regular expression to support subdomain matching.  If this list is empty CORS will not be enabled.")
 	fs.BoolVar(&s.AllowPrivileged, "allow-privileged", s.AllowPrivileged, "If true, allow privileged containers.")
+	fs.BoolVar(&s.EnableSELinuxIntegration, "selinux-integration", s.EnableSELinuxIntegration, "If true the api-server will accept pods which specify SELinux context. [default=false]")
```

Comment:
The word "integration" adds nothing to this flag or the variables.  Shorter is better.  It is "Go" after all and not "GooglesNewProgrammingLanguage"